src/components/Chart.tsx

```diff
@@ -89,6 +96,18 @@ export function Chart(props: ChartProps) {
   )
 }
 
+const blinkAnimation = keyframes`
+  0% { opacity: 100% }
+  50% { opacity: 50% }
+  100% { opacity: 100% }
+`
+
+const GBlink = styled.g`
```

Comment:
To avoid code duplication and rerendering of the `Node`, could we maybe make a single styled component that takes an argument whether to blink or not?

For example:
```typescript
// could be named G or StyledG or whatever
const G = styled.g<{ blink: boolean }>`
  animation-name: ${props => props.blink ? blinkAnimation : "none"};
  animation-duration: 2s;
  animation-iteration-count: infinite;
`
```
and then use it like 

```typescript
<G blink={clickedObject?.path === d.data.path}>
...
</G>
```